src/LaunchDarkly.ServerSdk/Files/FileDataSource.cs

```diff
@@ -84,7 +84,7 @@ private void LoadAll()
             {
                 try
                 {
-                    var content = File.ReadAllText(path);
+                    var content = ReadFileContent(path);

```

Comment:
Because of when the FileNotifier raises the change, the file is usually not done being written by the separate process when this code executes.  As a result, the file is still locked and `File.ReadAllText` fails.  Without retry, the change is missed.  :(

![image](https://user-images.githubusercontent.com/12101120/65919359-14895200-e3aa-11e9-8a08-47ac9f6098ad.png)

I find on my machine that it usually succeeds on the 2nd attempt, however, this would be machine and drive dependent:

![image](https://user-images.githubusercontent.com/12101120/65919445-4dc1c200-e3aa-11e9-8f35-1c796aaec431.png)